### PR TITLE
THEBIGBUGFIX: require config

### PR DESCRIPTION
**Pull request by taylortom**
Tidy RequireJS config

### CODE REVIEW

**Comment by dancgray** [comment]
@taylortom these changes are duplicated in THEBIGBUGFIX #2: some whitespace fixes https://github.com/adaptlearning/adapt_authoring/pull/1545

**Comment by taylortom** [comment]
Dang, that PR's messed up. Will look into it.

**Comment by taylortom** [comment]
@dancgray other PR should be sorted now

**Comment by taylortom** [comment]
@tomgreenfield done, and alphabetised everything.

File: frontend/src/core/app/config.js [hunk 1]
```diff
@@ -1,119 +1,116 @@
...
+    backbone: 'core/libraries/backbone',
+    backboneForms:'core/libraries/backbone-forms',
+    backboneFormsLists:'core/libraries/backbone-forms-lists',
+    colorPicker: 'core/libraries/colorPicker/js/colorpicker',
+    handlebars: 'core/libraries/handlebars',
+    imageReady: 'core/libraries/imageReady',
+    inview: 'core/libraries/inview',
+    jquery: 'core/libraries/jquery',
+    'jquery-ui': 'core/libraries/jquery-ui.min',
+    'jquery-form' : 'core/libraries/jquery.form',
```
Comment (tomgreenfield) [inline frontend/src/core/app/config.js]: `'jquery-ui'` & `'jquery-form'` need swapping if we're sorting by name.

File: frontend/src/core/app/config.js [hunk 2]
```diff
@@ -1,119 +1,116 @@
...
+    },
+    inview: {
+      deps: ['jquery'],
+      exports: 'inview'
+    },
+    'jquery-ui': {
+      deps: ['jquery'],
+      exports: "$"
+    },
+    'jquery-form': {
```
Comment (tomgreenfield) [inline frontend/src/core/app/config.js]: `'jquery-ui'` & `'jquery-form'` need swapping if we're sorting by name.

**Comment by taylortom** [comment]
Adding this to the structure PR

**Pull request closed without merging**